<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>installer</class>
 <widget class="QMainWindow" name="installer">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>491</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>原神启动器Ex安装程序</string>
  </property>
  <property name="windowIcon">
   <iconset resource="styple.qrc">
    <normaloff>:/ico/ico.ico</normaloff>:/ico/ico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Main_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>730</width>
      <height>470</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border:none;
}
QPushButton:hove {
	padding-bottom:5px;
}
QWidget{
	border-radius:5px;
}</string>
    </property>
    <widget class="QLabel" name="Fontground_Label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>730</width>
       <height>330</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/background.png);
border-bottom-right-radius:0px;
border-bottom-left-radius:0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Title_Lable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>原神启动器EX安装程序</string>
     </property>
    </widget>
    <widget class="QPushButton" name="InstallerStart_Button">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>240</y>
       <width>200</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(255, 200, 31);
	border-radius:5
}
QPushButton:pressed {
	background-color:rgb(185, 144, 22);
	border-radius:5

}</string>
     </property>
     <property name="text">
      <string>立即安装</string>
     </property>
    </widget>
    <widget class="QFrame" name="Bottom_Installer_Frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>730</width>
       <height>140</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-top-right-radius:0px;
border-top-left-radius:0px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="Installer_Frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>711</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border:none;
	border-radius:5px;
	
	color: rgb(244, 216, 135);
	background-color: rgb(57, 59, 64);
}
QLineEdit {
	
	border: 1px solid gray;
	border-radius: 5px;
	padding: 0 8px;
	selection-background-color: darkgray;
}
QFrame {
	border:none;
	}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Look_Button">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>40</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:pressed {
		
	background-color: rgb(34, 35, 38);
}</string>
       </property>
       <property name="text">
        <string>定位游戏</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Path_LineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>531</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="Installer_Label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>安装位置</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>256</width>
         <height>19</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QCheckBox" name="CreateStartedLink_CheckBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>创建桌面快捷方式</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CreateDesktopLink_CheckBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>创建开始快捷方式</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="Progress_Frame">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>-10</y>
         <width>731</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="Progress_Label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>381</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>正在安装中</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="Progress_ProgressBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>681</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar {
    min-height: 20px;
    max-height: 20px;
    border-radius: 5px;
	border: 2px solid rgb(57, 59, 64);
	text-align: center;
	background-color: rgba(255, 255, 255 ,0);
}
QProgressBar:chunk {
    border-radius: 3px;
    background-color: rgb(57, 59, 64);
    margin: 2px;
}

</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="StartLauncher_CheckBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>112</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>打开原神启动器Ex</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="InstallerEnd_Button">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>240</y>
       <width>200</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(255, 200, 31);
	border-radius:5
}
QPushButton:pressed {
	background-color:rgb(185, 144, 22);
	border-radius:5

}</string>
     </property>
     <property name="text">
      <string>安装完成</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QFrame" name="Top_Right_Frame">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>71</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Top_Right_Layout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="Min_Bottom">
         <property name="styleSheet">
          <string notr="true">QPushButton:hover {
	background-color: rgba(255, 255, 255 ,95);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="styple.qrc">
           <normaloff>:/ico/min.png</normaloff>:/ico/min.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Close_Buttom">
         <property name="styleSheet">
          <string notr="true">QPushButton:hover {
	background-color: rgba(255, 0, 0 ,95);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="styple.qrc">
           <normaloff>:/ico/close.png</normaloff>:/ico/close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <zorder>Fontground_Label</zorder>
    <zorder>Title_Lable</zorder>
    <zorder>Bottom_Installer_Frame</zorder>
    <zorder>InstallerStart_Button</zorder>
    <zorder>InstallerEnd_Button</zorder>
    <zorder>Top_Right_Frame</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="styple.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Min_Bottom</sender>
   <signal>clicked()</signal>
   <receiver>installer</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Close_Buttom</sender>
   <signal>clicked()</signal>
   <receiver>installer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>726</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
